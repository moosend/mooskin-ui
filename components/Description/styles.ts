import styled from 'styled-components';

// Models
import { IBoxComponentProps } from '../Box/model';

// Components
import { Box } from '../Box/Box';

// "CSS" variables
import variables from '../_utils/globals/variables';

export const StyledDescription = styled(Box)<IBoxComponentProps>`
	margin-top: 10px;
	font-size: 12px;
	color: ${(props) => props.palette?.fontColors.medgray1 || variables.fontColors.medgray1};
	width: fit-content;
`;

StyledDescription.displayName = 'StyledDescription';
