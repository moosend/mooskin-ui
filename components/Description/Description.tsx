import * as React from 'react';

// Mooskin Context HoC that passes context to component props
import { withMooskinContext } from '../Styled/MooskinContextProvider';

// Models
import { IBoxComponentProps } from '../Box/model';

// Styled Components
import { StyledDescription } from './styles';

/**
 * Description
 */
export const Description: React.FC<IBoxComponentProps> = withMooskinContext((props) => {
	return <StyledDescription {...props} />;
});

Description.defaultProps = {
	className: '',
	style: {},
};

Description.displayName = 'Description';
