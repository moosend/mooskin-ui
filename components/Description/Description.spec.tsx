import * as React from 'react';
import { Description } from './Description';

import { mount } from 'enzyme';

describe('Description', () => {
	test('renders correctly', () => {
		const tree = mount(<Description>Description here!</Description>);
		expect(tree).toMatchSnapshot();
	});
});
