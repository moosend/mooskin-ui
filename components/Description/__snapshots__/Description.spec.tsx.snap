// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description renders correctly 1`] = `
<Description
  className=""
  style={Object {}}
>
  <Component
    className=""
    palette={
      Object {
        "backgroundColors": Object {
          "accountNavigation": "#FFFFFF",
          "altWhite": "#FFFFFF",
          "autosExpression": "#ffffff",
          "background": "#FAFAFA",
          "black": "#000000",
          "blue400": "#4299E1",
          "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
          "client": "#E0E0E0",
          "createNew": "#FFFFFF",
          "darkgray1": "#777777",
          "darkgray2": "#454545",
          "darkgray3": "#424242",
          "dropdown": "#FFFFFF",
          "gray1": "#E0E0E0",
          "gray100": "#F0F0F0",
          "gray2": "#E7E7E7",
          "grayAlpha": "rgba(0,0,0,0.17)",
          "grayAlpha100": "rgba(0,0,0,0.06)",
          "grayAlpha400": "rgba(0,0,0,0.36)",
          "green600": "#2F855A",
          "inverseText": "#212121",
          "medgray1": "#757575",
          "medgray2": "#888888",
          "menuDark": "#21262A",
          "menuLight": "#2E3841",
          "orange400": "#ED8936",
          "pink400": "#ED64A6",
          "primary1": "#5548D9",
          "primary2": "#212121",
          "primary3": "#00796B",
          "primary4": "#D32F2F",
          "primary5": "#f0b941",
          "purple400": "#9f7aea",
          "red400": "#f56565",
          "red600": "#C53030",
          "secondary1": "#2E7D32",
          "secondary2": "#757575",
          "secondary3": "#673AB7",
          "secondary4": "#9C27B0",
          "secondary5": "#00796B",
          "secondary6": "#ECF0FC",
          "submenu": "#FFFFFF",
          "teal400": "#38B2AC",
          "teal500": "#02999A",
          "text": "#212121",
          "white": "#FFFFFF",
        },
        "borderColors": Object {
          "accountNavigation": "#FFFFFF",
          "altWhite": "#FFFFFF",
          "autosExpression": "#ffffff",
          "background": "#FAFAFA",
          "black": "#000000",
          "blue400": "#4299E1",
          "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
          "client": "#E0E0E0",
          "createNew": "#FFFFFF",
          "darkgray1": "#777777",
          "darkgray2": "#454545",
          "darkgray3": "#424242",
          "dropdown": "#FFFFFF",
          "gray1": "#E0E0E0",
          "gray100": "#F0F0F0",
          "gray2": "#E7E7E7",
          "grayAlpha": "rgba(0,0,0,0.17)",
          "grayAlpha100": "rgba(0,0,0,0.06)",
          "grayAlpha400": "rgba(0,0,0,0.36)",
          "green600": "#2F855A",
          "inverseText": "#212121",
          "medgray1": "#757575",
          "medgray2": "#888888",
          "menuDark": "#21262A",
          "menuLight": "#2E3841",
          "orange400": "#ED8936",
          "pink400": "#ED64A6",
          "primary1": "#5548D9",
          "primary2": "#212121",
          "primary3": "#00796B",
          "primary4": "#D32F2F",
          "primary5": "#f0b941",
          "purple400": "#9f7aea",
          "red400": "#f56565",
          "red600": "#C53030",
          "secondary1": "#2E7D32",
          "secondary2": "#757575",
          "secondary3": "#673AB7",
          "secondary4": "#9C27B0",
          "secondary5": "#00796B",
          "secondary6": "#ECF0FC",
          "submenu": "#FFFFFF",
          "teal400": "#38B2AC",
          "teal500": "#02999A",
          "text": "#212121",
          "white": "#FFFFFF",
        },
        "fontColors": Object {
          "accountNavigation": "#FFFFFF",
          "altWhite": "#FFFFFF",
          "autosExpression": "#ffffff",
          "background": "#FAFAFA",
          "black": "#000000",
          "blue400": "#4299E1",
          "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
          "client": "#E0E0E0",
          "createNew": "#212121",
          "darkgray1": "#777777",
          "darkgray2": "#454545",
          "darkgray3": "#424242",
          "dropdown": "#212121",
          "gray1": "#E0E0E0",
          "gray100": "#F0F0F0",
          "gray2": "#E7E7E7",
          "grayAlpha": "rgba(0,0,0,0.17)",
          "grayAlpha100": "rgba(0,0,0,0.06)",
          "grayAlpha400": "rgba(0,0,0,0.36)",
          "green600": "#2F855A",
          "inverseText": "#212121",
          "medgray1": "#757575",
          "medgray2": "#888888",
          "menuDark": "#21262A",
          "menuLight": "#2E3841",
          "orange400": "#ED8936",
          "pink400": "#ED64A6",
          "primary1": "#5548D9",
          "primary2": "#212121",
          "primary3": "#00796B",
          "primary4": "#D32F2F",
          "primary5": "#f0b941",
          "purple400": "#9f7aea",
          "red400": "#f56565",
          "red600": "#C53030",
          "secondary1": "#2E7D32",
          "secondary2": "#757575",
          "secondary3": "#673AB7",
          "secondary4": "#9C27B0",
          "secondary5": "#00796B",
          "secondary6": "#ECF0FC",
          "submenu": "#FFFFFF",
          "teal400": "#38B2AC",
          "teal500": "#02999A",
          "text": "#212121",
          "white": "#FFFFFF",
        },
      }
    }
    style={Object {}}
  >
    <StyledDescription
      className=""
      palette={
        Object {
          "backgroundColors": Object {
            "accountNavigation": "#FFFFFF",
            "altWhite": "#FFFFFF",
            "autosExpression": "#ffffff",
            "background": "#FAFAFA",
            "black": "#000000",
            "blue400": "#4299E1",
            "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
            "client": "#E0E0E0",
            "createNew": "#FFFFFF",
            "darkgray1": "#777777",
            "darkgray2": "#454545",
            "darkgray3": "#424242",
            "dropdown": "#FFFFFF",
            "gray1": "#E0E0E0",
            "gray100": "#F0F0F0",
            "gray2": "#E7E7E7",
            "grayAlpha": "rgba(0,0,0,0.17)",
            "grayAlpha100": "rgba(0,0,0,0.06)",
            "grayAlpha400": "rgba(0,0,0,0.36)",
            "green600": "#2F855A",
            "inverseText": "#212121",
            "medgray1": "#757575",
            "medgray2": "#888888",
            "menuDark": "#21262A",
            "menuLight": "#2E3841",
            "orange400": "#ED8936",
            "pink400": "#ED64A6",
            "primary1": "#5548D9",
            "primary2": "#212121",
            "primary3": "#00796B",
            "primary4": "#D32F2F",
            "primary5": "#f0b941",
            "purple400": "#9f7aea",
            "red400": "#f56565",
            "red600": "#C53030",
            "secondary1": "#2E7D32",
            "secondary2": "#757575",
            "secondary3": "#673AB7",
            "secondary4": "#9C27B0",
            "secondary5": "#00796B",
            "secondary6": "#ECF0FC",
            "submenu": "#FFFFFF",
            "teal400": "#38B2AC",
            "teal500": "#02999A",
            "text": "#212121",
            "white": "#FFFFFF",
          },
          "borderColors": Object {
            "accountNavigation": "#FFFFFF",
            "altWhite": "#FFFFFF",
            "autosExpression": "#ffffff",
            "background": "#FAFAFA",
            "black": "#000000",
            "blue400": "#4299E1",
            "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
            "client": "#E0E0E0",
            "createNew": "#FFFFFF",
            "darkgray1": "#777777",
            "darkgray2": "#454545",
            "darkgray3": "#424242",
            "dropdown": "#FFFFFF",
            "gray1": "#E0E0E0",
            "gray100": "#F0F0F0",
            "gray2": "#E7E7E7",
            "grayAlpha": "rgba(0,0,0,0.17)",
            "grayAlpha100": "rgba(0,0,0,0.06)",
            "grayAlpha400": "rgba(0,0,0,0.36)",
            "green600": "#2F855A",
            "inverseText": "#212121",
            "medgray1": "#757575",
            "medgray2": "#888888",
            "menuDark": "#21262A",
            "menuLight": "#2E3841",
            "orange400": "#ED8936",
            "pink400": "#ED64A6",
            "primary1": "#5548D9",
            "primary2": "#212121",
            "primary3": "#00796B",
            "primary4": "#D32F2F",
            "primary5": "#f0b941",
            "purple400": "#9f7aea",
            "red400": "#f56565",
            "red600": "#C53030",
            "secondary1": "#2E7D32",
            "secondary2": "#757575",
            "secondary3": "#673AB7",
            "secondary4": "#9C27B0",
            "secondary5": "#00796B",
            "secondary6": "#ECF0FC",
            "submenu": "#FFFFFF",
            "teal400": "#38B2AC",
            "teal500": "#02999A",
            "text": "#212121",
            "white": "#FFFFFF",
          },
          "fontColors": Object {
            "accountNavigation": "#FFFFFF",
            "altWhite": "#FFFFFF",
            "autosExpression": "#ffffff",
            "background": "#FAFAFA",
            "black": "#000000",
            "blue400": "#4299E1",
            "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
            "client": "#E0E0E0",
            "createNew": "#212121",
            "darkgray1": "#777777",
            "darkgray2": "#454545",
            "darkgray3": "#424242",
            "dropdown": "#212121",
            "gray1": "#E0E0E0",
            "gray100": "#F0F0F0",
            "gray2": "#E7E7E7",
            "grayAlpha": "rgba(0,0,0,0.17)",
            "grayAlpha100": "rgba(0,0,0,0.06)",
            "grayAlpha400": "rgba(0,0,0,0.36)",
            "green600": "#2F855A",
            "inverseText": "#212121",
            "medgray1": "#757575",
            "medgray2": "#888888",
            "menuDark": "#21262A",
            "menuLight": "#2E3841",
            "orange400": "#ED8936",
            "pink400": "#ED64A6",
            "primary1": "#5548D9",
            "primary2": "#212121",
            "primary3": "#00796B",
            "primary4": "#D32F2F",
            "primary5": "#f0b941",
            "purple400": "#9f7aea",
            "red400": "#f56565",
            "red600": "#C53030",
            "secondary1": "#2E7D32",
            "secondary2": "#757575",
            "secondary3": "#673AB7",
            "secondary4": "#9C27B0",
            "secondary5": "#00796B",
            "secondary6": "#ECF0FC",
            "submenu": "#FFFFFF",
            "teal400": "#38B2AC",
            "teal500": "#02999A",
            "text": "#212121",
            "white": "#FFFFFF",
          },
        }
      }
      style={Object {}}
    >
      <Box
        className="sc-gsDKAQ fUvHql"
        palette={
          Object {
            "backgroundColors": Object {
              "accountNavigation": "#FFFFFF",
              "altWhite": "#FFFFFF",
              "autosExpression": "#ffffff",
              "background": "#FAFAFA",
              "black": "#000000",
              "blue400": "#4299E1",
              "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
              "client": "#E0E0E0",
              "createNew": "#FFFFFF",
              "darkgray1": "#777777",
              "darkgray2": "#454545",
              "darkgray3": "#424242",
              "dropdown": "#FFFFFF",
              "gray1": "#E0E0E0",
              "gray100": "#F0F0F0",
              "gray2": "#E7E7E7",
              "grayAlpha": "rgba(0,0,0,0.17)",
              "grayAlpha100": "rgba(0,0,0,0.06)",
              "grayAlpha400": "rgba(0,0,0,0.36)",
              "green600": "#2F855A",
              "inverseText": "#212121",
              "medgray1": "#757575",
              "medgray2": "#888888",
              "menuDark": "#21262A",
              "menuLight": "#2E3841",
              "orange400": "#ED8936",
              "pink400": "#ED64A6",
              "primary1": "#5548D9",
              "primary2": "#212121",
              "primary3": "#00796B",
              "primary4": "#D32F2F",
              "primary5": "#f0b941",
              "purple400": "#9f7aea",
              "red400": "#f56565",
              "red600": "#C53030",
              "secondary1": "#2E7D32",
              "secondary2": "#757575",
              "secondary3": "#673AB7",
              "secondary4": "#9C27B0",
              "secondary5": "#00796B",
              "secondary6": "#ECF0FC",
              "submenu": "#FFFFFF",
              "teal400": "#38B2AC",
              "teal500": "#02999A",
              "text": "#212121",
              "white": "#FFFFFF",
            },
            "borderColors": Object {
              "accountNavigation": "#FFFFFF",
              "altWhite": "#FFFFFF",
              "autosExpression": "#ffffff",
              "background": "#FAFAFA",
              "black": "#000000",
              "blue400": "#4299E1",
              "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
              "client": "#E0E0E0",
              "createNew": "#FFFFFF",
              "darkgray1": "#777777",
              "darkgray2": "#454545",
              "darkgray3": "#424242",
              "dropdown": "#FFFFFF",
              "gray1": "#E0E0E0",
              "gray100": "#F0F0F0",
              "gray2": "#E7E7E7",
              "grayAlpha": "rgba(0,0,0,0.17)",
              "grayAlpha100": "rgba(0,0,0,0.06)",
              "grayAlpha400": "rgba(0,0,0,0.36)",
              "green600": "#2F855A",
              "inverseText": "#212121",
              "medgray1": "#757575",
              "medgray2": "#888888",
              "menuDark": "#21262A",
              "menuLight": "#2E3841",
              "orange400": "#ED8936",
              "pink400": "#ED64A6",
              "primary1": "#5548D9",
              "primary2": "#212121",
              "primary3": "#00796B",
              "primary4": "#D32F2F",
              "primary5": "#f0b941",
              "purple400": "#9f7aea",
              "red400": "#f56565",
              "red600": "#C53030",
              "secondary1": "#2E7D32",
              "secondary2": "#757575",
              "secondary3": "#673AB7",
              "secondary4": "#9C27B0",
              "secondary5": "#00796B",
              "secondary6": "#ECF0FC",
              "submenu": "#FFFFFF",
              "teal400": "#38B2AC",
              "teal500": "#02999A",
              "text": "#212121",
              "white": "#FFFFFF",
            },
            "fontColors": Object {
              "accountNavigation": "#FFFFFF",
              "altWhite": "#FFFFFF",
              "autosExpression": "#ffffff",
              "background": "#FAFAFA",
              "black": "#000000",
              "blue400": "#4299E1",
              "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
              "client": "#E0E0E0",
              "createNew": "#212121",
              "darkgray1": "#777777",
              "darkgray2": "#454545",
              "darkgray3": "#424242",
              "dropdown": "#212121",
              "gray1": "#E0E0E0",
              "gray100": "#F0F0F0",
              "gray2": "#E7E7E7",
              "grayAlpha": "rgba(0,0,0,0.17)",
              "grayAlpha100": "rgba(0,0,0,0.06)",
              "grayAlpha400": "rgba(0,0,0,0.36)",
              "green600": "#2F855A",
              "inverseText": "#212121",
              "medgray1": "#757575",
              "medgray2": "#888888",
              "menuDark": "#21262A",
              "menuLight": "#2E3841",
              "orange400": "#ED8936",
              "pink400": "#ED64A6",
              "primary1": "#5548D9",
              "primary2": "#212121",
              "primary3": "#00796B",
              "primary4": "#D32F2F",
              "primary5": "#f0b941",
              "purple400": "#9f7aea",
              "red400": "#f56565",
              "red600": "#C53030",
              "secondary1": "#2E7D32",
              "secondary2": "#757575",
              "secondary3": "#673AB7",
              "secondary4": "#9C27B0",
              "secondary5": "#00796B",
              "secondary6": "#ECF0FC",
              "submenu": "#FFFFFF",
              "teal400": "#38B2AC",
              "teal500": "#02999A",
              "text": "#212121",
              "white": "#FFFFFF",
            },
          }
        }
        style={Object {}}
        transition="0.2s all"
      >
        <Component
          className="sc-gsDKAQ fUvHql"
          palette={
            Object {
              "backgroundColors": Object {
                "accountNavigation": "#FFFFFF",
                "altWhite": "#FFFFFF",
                "autosExpression": "#ffffff",
                "background": "#FAFAFA",
                "black": "#000000",
                "blue400": "#4299E1",
                "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
                "client": "#E0E0E0",
                "createNew": "#FFFFFF",
                "darkgray1": "#777777",
                "darkgray2": "#454545",
                "darkgray3": "#424242",
                "dropdown": "#FFFFFF",
                "gray1": "#E0E0E0",
                "gray100": "#F0F0F0",
                "gray2": "#E7E7E7",
                "grayAlpha": "rgba(0,0,0,0.17)",
                "grayAlpha100": "rgba(0,0,0,0.06)",
                "grayAlpha400": "rgba(0,0,0,0.36)",
                "green600": "#2F855A",
                "inverseText": "#212121",
                "medgray1": "#757575",
                "medgray2": "#888888",
                "menuDark": "#21262A",
                "menuLight": "#2E3841",
                "orange400": "#ED8936",
                "pink400": "#ED64A6",
                "primary1": "#5548D9",
                "primary2": "#212121",
                "primary3": "#00796B",
                "primary4": "#D32F2F",
                "primary5": "#f0b941",
                "purple400": "#9f7aea",
                "red400": "#f56565",
                "red600": "#C53030",
                "secondary1": "#2E7D32",
                "secondary2": "#757575",
                "secondary3": "#673AB7",
                "secondary4": "#9C27B0",
                "secondary5": "#00796B",
                "secondary6": "#ECF0FC",
                "submenu": "#FFFFFF",
                "teal400": "#38B2AC",
                "teal500": "#02999A",
                "text": "#212121",
                "white": "#FFFFFF",
              },
              "borderColors": Object {
                "accountNavigation": "#FFFFFF",
                "altWhite": "#FFFFFF",
                "autosExpression": "#ffffff",
                "background": "#FAFAFA",
                "black": "#000000",
                "blue400": "#4299E1",
                "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
                "client": "#E0E0E0",
                "createNew": "#FFFFFF",
                "darkgray1": "#777777",
                "darkgray2": "#454545",
                "darkgray3": "#424242",
                "dropdown": "#FFFFFF",
                "gray1": "#E0E0E0",
                "gray100": "#F0F0F0",
                "gray2": "#E7E7E7",
                "grayAlpha": "rgba(0,0,0,0.17)",
                "grayAlpha100": "rgba(0,0,0,0.06)",
                "grayAlpha400": "rgba(0,0,0,0.36)",
                "green600": "#2F855A",
                "inverseText": "#212121",
                "medgray1": "#757575",
                "medgray2": "#888888",
                "menuDark": "#21262A",
                "menuLight": "#2E3841",
                "orange400": "#ED8936",
                "pink400": "#ED64A6",
                "primary1": "#5548D9",
                "primary2": "#212121",
                "primary3": "#00796B",
                "primary4": "#D32F2F",
                "primary5": "#f0b941",
                "purple400": "#9f7aea",
                "red400": "#f56565",
                "red600": "#C53030",
                "secondary1": "#2E7D32",
                "secondary2": "#757575",
                "secondary3": "#673AB7",
                "secondary4": "#9C27B0",
                "secondary5": "#00796B",
                "secondary6": "#ECF0FC",
                "submenu": "#FFFFFF",
                "teal400": "#38B2AC",
                "teal500": "#02999A",
                "text": "#212121",
                "white": "#FFFFFF",
              },
              "fontColors": Object {
                "accountNavigation": "#FFFFFF",
                "altWhite": "#FFFFFF",
                "autosExpression": "#ffffff",
                "background": "#FAFAFA",
                "black": "#000000",
                "blue400": "#4299E1",
                "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
                "client": "#E0E0E0",
                "createNew": "#212121",
                "darkgray1": "#777777",
                "darkgray2": "#454545",
                "darkgray3": "#424242",
                "dropdown": "#212121",
                "gray1": "#E0E0E0",
                "gray100": "#F0F0F0",
                "gray2": "#E7E7E7",
                "grayAlpha": "rgba(0,0,0,0.17)",
                "grayAlpha100": "rgba(0,0,0,0.06)",
                "grayAlpha400": "rgba(0,0,0,0.36)",
                "green600": "#2F855A",
                "inverseText": "#212121",
                "medgray1": "#757575",
                "medgray2": "#888888",
                "menuDark": "#21262A",
                "menuLight": "#2E3841",
                "orange400": "#ED8936",
                "pink400": "#ED64A6",
                "primary1": "#5548D9",
                "primary2": "#212121",
                "primary3": "#00796B",
                "primary4": "#D32F2F",
                "primary5": "#f0b941",
                "purple400": "#9f7aea",
                "red400": "#f56565",
                "red600": "#C53030",
                "secondary1": "#2E7D32",
                "secondary2": "#757575",
                "secondary3": "#673AB7",
                "secondary4": "#9C27B0",
                "secondary5": "#00796B",
                "secondary6": "#ECF0FC",
                "submenu": "#FFFFFF",
                "teal400": "#38B2AC",
                "teal500": "#02999A",
                "text": "#212121",
                "white": "#FFFFFF",
              },
            }
          }
          style={Object {}}
          transition="0.2s all"
        >
          <StyledBox
            className="sc-gsDKAQ fUvHql"
            palette={
              Object {
                "backgroundColors": Object {
                  "accountNavigation": "#FFFFFF",
                  "altWhite": "#FFFFFF",
                  "autosExpression": "#ffffff",
                  "background": "#FAFAFA",
                  "black": "#000000",
                  "blue400": "#4299E1",
                  "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
                  "client": "#E0E0E0",
                  "createNew": "#FFFFFF",
                  "darkgray1": "#777777",
                  "darkgray2": "#454545",
                  "darkgray3": "#424242",
                  "dropdown": "#FFFFFF",
                  "gray1": "#E0E0E0",
                  "gray100": "#F0F0F0",
                  "gray2": "#E7E7E7",
                  "grayAlpha": "rgba(0,0,0,0.17)",
                  "grayAlpha100": "rgba(0,0,0,0.06)",
                  "grayAlpha400": "rgba(0,0,0,0.36)",
                  "green600": "#2F855A",
                  "inverseText": "#212121",
                  "medgray1": "#757575",
                  "medgray2": "#888888",
                  "menuDark": "#21262A",
                  "menuLight": "#2E3841",
                  "orange400": "#ED8936",
                  "pink400": "#ED64A6",
                  "primary1": "#5548D9",
                  "primary2": "#212121",
                  "primary3": "#00796B",
                  "primary4": "#D32F2F",
                  "primary5": "#f0b941",
                  "purple400": "#9f7aea",
                  "red400": "#f56565",
                  "red600": "#C53030",
                  "secondary1": "#2E7D32",
                  "secondary2": "#757575",
                  "secondary3": "#673AB7",
                  "secondary4": "#9C27B0",
                  "secondary5": "#00796B",
                  "secondary6": "#ECF0FC",
                  "submenu": "#FFFFFF",
                  "teal400": "#38B2AC",
                  "teal500": "#02999A",
                  "text": "#212121",
                  "white": "#FFFFFF",
                },
                "borderColors": Object {
                  "accountNavigation": "#FFFFFF",
                  "altWhite": "#FFFFFF",
                  "autosExpression": "#ffffff",
                  "background": "#FAFAFA",
                  "black": "#000000",
                  "blue400": "#4299E1",
                  "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
                  "client": "#E0E0E0",
                  "createNew": "#FFFFFF",
                  "darkgray1": "#777777",
                  "darkgray2": "#454545",
                  "darkgray3": "#424242",
                  "dropdown": "#FFFFFF",
                  "gray1": "#E0E0E0",
                  "gray100": "#F0F0F0",
                  "gray2": "#E7E7E7",
                  "grayAlpha": "rgba(0,0,0,0.17)",
                  "grayAlpha100": "rgba(0,0,0,0.06)",
                  "grayAlpha400": "rgba(0,0,0,0.36)",
                  "green600": "#2F855A",
                  "inverseText": "#212121",
                  "medgray1": "#757575",
                  "medgray2": "#888888",
                  "menuDark": "#21262A",
                  "menuLight": "#2E3841",
                  "orange400": "#ED8936",
                  "pink400": "#ED64A6",
                  "primary1": "#5548D9",
                  "primary2": "#212121",
                  "primary3": "#00796B",
                  "primary4": "#D32F2F",
                  "primary5": "#f0b941",
                  "purple400": "#9f7aea",
                  "red400": "#f56565",
                  "red600": "#C53030",
                  "secondary1": "#2E7D32",
                  "secondary2": "#757575",
                  "secondary3": "#673AB7",
                  "secondary4": "#9C27B0",
                  "secondary5": "#00796B",
                  "secondary6": "#ECF0FC",
                  "submenu": "#FFFFFF",
                  "teal400": "#38B2AC",
                  "teal500": "#02999A",
                  "text": "#212121",
                  "white": "#FFFFFF",
                },
                "fontColors": Object {
                  "accountNavigation": "#FFFFFF",
                  "altWhite": "#FFFFFF",
                  "autosExpression": "#ffffff",
                  "background": "#FAFAFA",
                  "black": "#000000",
                  "blue400": "#4299E1",
                  "checkboxUnselected": "rgba(0, 0, 0, 0.62)",
                  "client": "#E0E0E0",
                  "createNew": "#212121",
                  "darkgray1": "#777777",
                  "darkgray2": "#454545",
                  "darkgray3": "#424242",
                  "dropdown": "#212121",
                  "gray1": "#E0E0E0",
                  "gray100": "#F0F0F0",
                  "gray2": "#E7E7E7",
                  "grayAlpha": "rgba(0,0,0,0.17)",
                  "grayAlpha100": "rgba(0,0,0,0.06)",
                  "grayAlpha400": "rgba(0,0,0,0.36)",
                  "green600": "#2F855A",
                  "inverseText": "#212121",
                  "medgray1": "#757575",
                  "medgray2": "#888888",
                  "menuDark": "#21262A",
                  "menuLight": "#2E3841",
                  "orange400": "#ED8936",
                  "pink400": "#ED64A6",
                  "primary1": "#5548D9",
                  "primary2": "#212121",
                  "primary3": "#00796B",
                  "primary4": "#D32F2F",
                  "primary5": "#f0b941",
                  "purple400": "#9f7aea",
                  "red400": "#f56565",
                  "red600": "#C53030",
                  "secondary1": "#2E7D32",
                  "secondary2": "#757575",
                  "secondary3": "#673AB7",
                  "secondary4": "#9C27B0",
                  "secondary5": "#00796B",
                  "secondary6": "#ECF0FC",
                  "submenu": "#FFFFFF",
                  "teal400": "#38B2AC",
                  "teal500": "#02999A",
                  "text": "#212121",
                  "white": "#FFFFFF",
                },
              }
            }
            style={Object {}}
            transition="0.2s all"
          >
            <div
              className="sc-bdvvtL buKeIX sc-gsDKAQ fUvHql"
              style={Object {}}
            >
              Description here!
            </div>
          </StyledBox>
        </Component>
      </Box>
    </StyledDescription>
  </Component>
</Description>
`;
